app/views/sessions/new.html.erb

```diff
@@ -0,0 +1,17 @@
+<% provide :title, "Log in" %>
+<h1>Log in</h1>
+
+  <div class="row">
+    <div class="col-md-6 col-md-offset-3">
+      <%= form_for(:session, url: login_path) do |f| %>
+
+        <%= f.label :email %>
+        <%= f.email_field :email, class: 'form-control' %>
+        <%= f.label :password %>
+        <%= f.password_field :password, class: 'form-control' %>
+        <%= f.submit "Log in", class: "btn btn-primary" %>
+
+    <% end %>
+    <p>New user? <%= link_to "Sign up now!", signup_path %></p>
+  </div>
+</div
```

Comment:
semilar

Reply:
Sorry for missing the newline,was spending time to check indent